nixos/modules/misc/version.nix

```diff
@@ -112,6 +154,8 @@ in
         HOME_URL="https://nixos.org/"
         SUPPORT_URL="https://nixos.org/nixos/support.html"
         BUG_REPORT_URL="https://github.com/NixOS/nixpkgs/issues"
+        IS_SUPPORTED=${toString config.system.nixosState.isSupported}
+        SUPPORT_STATE="${toString config.system.nixosState.state}"
```

Comment:
[This isn't part of `os-release` it seems](https://www.freedesktop.org/software/systemd/man/os-release.html).

They recommend prefixing new fields:

> Operating system vendors may extend the file format and introduce new fields. It is highly recommended to prefix new fields with an OS specific name in order to avoid name clashes. Applications reading this file must ignore unknown fields. Example: "DEBIAN_BTS="debbugs://bugs.debian.org/""

Reply:
Also, `toString true` -> "1". You probably want to use `lib.boolToString true` -> "true" instead.

Reply:
I think these changes are superfluous until some tool starts needing them.